react-exercises/grocery-list-react/src/App.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+import { GroceryList } from '../src/components/grocery-list/grocery-list.components';
+import { AddGroceryForm } from './components/add-grocery/add-grocery.component';
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      value: '',
+      groceries: [
+        { name: 'Basmati Rice', count: 1, color: 'black' },
+        { name: 'Butter', count: 1, color: 'black' },
+        { name: 'Milk', count: 1, color: 'black' },
+        { name: 'Curd', count: 1, color: 'black' },
+      ]
+    };
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.removeItem = this.removeItem.bind(this);
+    this.updateColor = this.updateColor.bind(this);
+    // this.removeSingleItem = this.removeSingleItem.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ value: event.target.value });
+  }
+
+  handleSubmit(event) {
+    let { groceries } = this.state;
+    const newGrocery = this.state.value.trim();
```

Comment:
The word `Grocery` itself is used attributively. Grocery list, grocery item etc.
```suggestion
    const newItem = this.state.value.trim();
```
OR
```suggestion
    const newGroceryItem = this.state.value.trim();
```